int64 a_operation_dir_in_deg
---
int64 a_operation_status
